src/funcx_common/task_storage/s3.py

```diff
@@ -13,6 +14,11 @@
 from .redis import ImplicitRedisStorage
 
 
+class Kind(Enum):
```

Comment:
`Kind` is very generic. Can we give this a more precise name?
`class StorageFieldName(Enum): ...`, for example.

Reply:
I can't think of a better name. `Type` is too overloaded. `flavor` doesn't make sense.